numpyro/distributions/flows.py

```diff
@@ -49,25 +51,34 @@ def __call__(self, x):
         mean, log_scale = out[..., 0, :], out[..., 1, :]
```

Comment:
note that this `0/1` stuff i wrote is kinda gross. it'd be better if shapes were such that we could just do
`mean, log_scale = self.arn.apply_fun(self.params, x)`
of course, you needn't do that in this PR. but maybe put a TODO?

Reply:
nice suggestion! I'll do it shortly in a follow-up PR.